src/ast/ast_annotations_ctx_collection.h

```diff
@@ -0,0 +1,41 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#pragma once
+#include "cypher-parser.h"
+
+typedef cypher_ast_annotation_context_t AnnotationCtx;
+
+typedef struct {
```

Comment:
Please add a short comment describing this struct.

Reply:
done